ports/stm32/boards/NUCLEO_WL55/manifest.py

```diff
@@ -0,0 +1,2 @@
+include("$(MPY_DIR)/extmod/uasyncio/manifest.py")
+freeze("$(MPY_LIB_DIR)/micropython/lora", ("async_modem.py", "stm32wl5.py"))
```

Comment:
Pros/cons for including this modem by default:

* Pro: Likely anyone using this board will be using a LoRa modem, as why else would you have it?
* Pro: stm32wl5.py source code is too big to fit into the small RAM on this device, needs to always be loaded as bytecode. Freezing it in makes it easier for users to get it working in the device.
* Con: Makes default image larger (+15KB, text size 219504->235104, of which about 4.5KB is LoRa modem and the rest if asyncio). 
* Con: Assumes user will want this modem and not an alternative modem implementation or version.

Can easily drop this commit if it's too much overhead.

A midway compromise could be to freeze in the synchronous modem driver stm32wl5.py (approx 3.5KB) but not the async support.

Reply:
I think we should freeze it in by default, because of your first point (why else would you use this board?).  And asyncio is so useful that we should also include that.

There are things we can do to reduce firmware size in other ways by removing other features that are less used (but that can be done later if needed).